test/matlab/TestImport.m

```diff
@@ -4,8 +4,8 @@
             self = self@TestCase(name);
         end
 
-        function testImportXML(self)
-            gas = Solution('gri30.xml', 'gri30_mix');
+        function testImportYAML(self)
```

Comment:
As @speth noted in #768, we shouldn't necessarily remove tests of the XML interface just yet. It's probably worth adding a separate test of the YAML interface here

Reply:
Adding a separate test here.